apps/web/src/components/common/layout/authenticated-layout.tsx

```diff
@@ -0,0 +1,26 @@
+import { ReactNode } from "react";
+
+import useUser from "lib/use-auth.hook";
+
+import NavigationBar from "../navigation-bar";
+
+type AuthenticatedLayoutProps = {
+  children: ReactNode;
+};
+
+export function AuthenticatedLayout({ children }: AuthenticatedLayoutProps) {
+  const { signOut } = useUser({
+    redirectTo: "/login",
+  });
+
+  return (
+    <main className={"min-h-screen bg-slate-50 flex flex-1"}>
+      <div className="h-full w-full">
+        <NavigationBar signOut={signOut} />
+        <article className="mx-auto max-w-7xl py-6 px-6 sm:px-6 lg:px-8">
```

Comment:
I was initially unsure about the nesting of `<section/>` instead `<article/>` but it seems to be semantically correct: https://www.w3schools.com/html/html5_semantic_elements.asp#:~:text=Nesting%20%3Carticle%3E%20in%20%3Csection%3E%20or%20Vice%20Versa%3F

Reply:
I don't know until now that it's controversial also 😆 
https://stackoverflow.com/questions/9527378/should-sections-have-articles-or-should-articles-have-sections#:~:text=It's%20entirely%20acceptable%20to%20nest,they%20are%20two%20different%20things.

My thinking when I used this was a page can have one or many articles, and each of the articles can have many sections, so it seems correct to nest `section` in `article`